Simplified/Accounts/Library/Account.swift

```diff
@@ -234,18 +234,17 @@ class OPDS2SamlIDP: NSObject, Codable {
       if trimmedUrlStr.lowercased().hasPrefix("nypl.card-creator:") {
         let cartCreatorUrlStr = String(trimmedUrlStr.dropFirst("nypl.card-creator:".count))
         signUpUrl = URL(string: cartCreatorUrlStr)
-        supportsCardCreator = (signUpUrl != nil)
```

Comment:
I think the intention here was to use the `signUpUrl` as a way to tell whether the library (not the user)  can support card creation. And I am not 100% sure but I _think_ `supportsCardCreator` actually means "supports native card creator"... something to clarify with Leonard maybe.

Reply:
fixed